packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -10,11 +10,13 @@ use hash_engine_lib::{
 
 use crate::process;
 
+const PROCESS_PATH_DEFAULT: &str = env!("CARGO_BIN_FILE_HASH_ENGINE");
```

Comment:
Can we rename these to `ENGINE_PATH_DEFAULT` or `ENGINE_BIN_PATH_...` 

Github won't let me suggest changes